examples/visual_gen/README.md

```
@@ -226,27 +276,31 @@ python visual_gen_ltx2.py \
-| `--seed` | ✓ | ✓ | ✓ | 42 | Random seed |
-| `--image` | — | ✓ | None | Input image for image-to-video |
-| `--image_cond_strength` | — | ✓ | 1.0 | Image conditioning strength |
-| `--enable_teacache` | ✓ | ✓ | — | False | Cache optimization |
-| `--teacache_thresh` | ✓ | ✓ | — | 0.2 | TeaCache similarity threshold |
-| `--attention_backend` | ✓ | ✓ | — | VANILLA | `VANILLA`, `TRTLLM`, or `FA4` |
-| `--cfg_size` | — | ✓ | — | 1 | CFG parallelism |
-| `--ulysses_size` | ✓ | ✓ | — | 1 | Sequence parallelism |
-| `--linear_type` | ✓ | ✓ | — | default | Quantization type |
+| Argument | FLUX | Qwen-Image | WAN | LTX2 | Default | Description |
+|----------|------|------------|-----|------|---------|-------------|
+| `--model_path` | ✓ | ✓ | ✓ | — | Path to model checkpoint directory |
```

markdownlint-cli2 (0.22.1)

[warning] Table column count
Expected: 7; Actual: 6; Too few cells, row will be missing data

(MD056, table-column-count)